float64[] currentFlangePostion
float64[] currentFlangePostionRight
float64[] axisCurrentAngle
float64[] axisCurrentAngleRight